common/include/kmx_file.h

```diff
@@ -302,6 +302,7 @@ namespace kmx {
 
 #define K_MODIFIERFLAG  0x007F
 #define K_NOTMODIFIERFLAG 0xFF00   // I4548
+#define K_DEFAULTMODFLAG  0x10000  // used by KMX+ for the default modifier
```

Comment:
Given this cannot be used in this context, as modifier flags are 16 bit in .kmx, I think this should be a comment. Also, suggest using full word rather than MOD

```suggestion
// Note: DEFAULT_MODIFIER = 0x10000, used by KMX+ for the 
// default modifier flag in layers, > 16 bit so not available here
```